src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```diff
@@ -184,6 +184,7 @@ $sidebar-width: 200px;
         display: block;
         font-size: 1.1em;
         padding: 10px;
+        padding-left: 25px;
```

Comment:
You need to ident the submenus, too.

Before your PR:
![before](https://user-images.githubusercontent.com/1897962/87955456-11cd2a00-caae-11ea-84ca-e9bad37a4cba.png)

After your PR:
![after](https://user-images.githubusercontent.com/1897962/87955506-26112700-caae-11ea-9436-df28b165312f.png)

Reply:
Thank you @votdev Updated the PR.
![navbar-paddding](https://user-images.githubusercontent.com/53651462/88023237-4d1f3580-cb4e-11ea-8e28-85596a784dff.png)